lib/src/common/js_proxy.dart

```diff
@@ -45,10 +45,12 @@ JsObject _buildJsProxy(JsProxy instance) {
   return proxy;
 }
 
-/// Query for all public fields/methods.
-final _queryOptions =
-    new smoke.QueryOptions(includeMethods: true, includeUpTo: HtmlElement,
-        excludeOverriden: true);
+class JsProxyReflectable extends Reflectable {
```

Comment:
This is actually used by many things, and should probably be renamed, but I haven't come up with a good name yet. Suggestions welcome ;).